core/config/docs/chains-evm.toml

```diff
@@ -112,7 +112,7 @@ Enabled = true # Default
 #
 # - `FixedPrice` uses static configured values for gas price (can be set via API call).
 # - `BlockHistory` dynamically adjusts default gas price based on heuristics from mined blocks.
-# - `L2Suggested` is a special mode only for use with L2 blockchains. This mode will use the gas price suggested by the rpc endpoint via `eth_gasPrice`.
```

Comment:
Rather than deleting it entirely, I would for now, say that this name is deprecated, and replaced by new name SuggestedPrice.

This will help someone who remembers the old name, know what happened to it.

Reply:
Sounds good